PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/enforcer/AzureSdkEnforcer.java

```diff
@@ -62,12 +63,15 @@ private static void warnDeprecatedLibs(@AzureTelemetry.Property List<? extends A
     private static String buildMessage(@Nonnull List<? extends AzureJavaSdkEntity> libs) {
         final String liPackages = libs.stream().map(l -> {
             if (StringUtils.isNotBlank(l.getReplace())) {
-                final String replacePackage = l.getReplace().trim();
+                final String[] replacePackages = l.getReplace().trim().split(",");
+                final String replaces = Arrays.stream(replacePackages)
+                    .map(p -> String.format("<a href='%s'>%s</a>", getMavenArtifactUrl(p.trim()), p.trim()))
```

Comment:
do we need to convert replace artifact to HTML list?

Reply:
yes

Reply:
no need.